python/pyspark/sql/functions.py

```diff
@@ -809,6 +809,45 @@ def ntile(n):
     return Column(sc._jvm.functions.ntile(int(n)))
 
 
+@since(2.3)
+def unboundedPreceding():
+    """
+    Window function: returns the special frame boundary that represents the first row
+    in the window partition.
+    >>> df = spark.createDataFrame([(5,)])
+    >>> df.select(unboundedPreceding()).show
+    <bound method DataFrame.show of DataFrame[UNBOUNDED PRECEDING: null]>
+    """
+    sc = SparkContext._active_spark_context
+    return Column(sc._jvm.functions.unboundedPreceding())
+
+
+@since(2.3)
+def unboundedFollowing():
+    """
+    Window function: returns the special frame boundary that represents the last row
+    in the window partition.
+    >>> df = spark.createDataFrame([(5,)])
```

Comment:
I believe we didn't claim we follow PEP 257 yet but I believe it would be good to have a newline between doctest and the description at least, if you don't mind.

Reply:
Will add a newline. Thanks.